IM_calculation/IM/Burks_Baker_2013_elastic_inelastic.py

```diff
@@ -132,7 +134,9 @@ def Bilinear_Newmark_withTH(
     # Ha = a
     # Hfs = fs
 
-    return d  # return all displacements, not just the maximum Sd. This is to compute rotd if needed
+    return (
+        d
+    )  # return all displacements, not just the maximum Sd. This is to compute rotd if needed
```

Comment:
This seems to be failing the black test

Reply:
It seems to think my black is not black enough - fixed.